python/pyarrow/table.pxi

```diff
@@ -140,7 +140,8 @@ cdef class ChunkedArray:
         return result
 
     def to_pandas(self, bint strings_to_categorical=False,
-                  bint zero_copy_only=False, bint integer_object_nulls=False):
```

Comment:
Do not check in python/pyarrow/plasma_store_server -- add to .gitignore

Reply:
Ouch

Reply:
Done